proto/viam/app/data/v1/data.proto

```diff
@@ -269,3 +296,31 @@ message TagsByFilterRequest {
 message TagsByFilterResponse {
   repeated string tags = 1;
 }
+
+message AddBoundingBoxToImageByIDRequest {
+  string file_id = 1;
+  string label = 2;
+  float normalized_x_min = 3;
+  float normalized_y_min = 4;
+  float normalized_x_max = 5;
+  float normalized_y_max = 6;
+}
+
+message AddBoundingBoxToImageByIDResponse {
+  string bounding_box_id = 1;
```

Comment:
Thoughts on this switch to make it more readable/explicit vs bbox_id which is common ML shorthand?

Reply:
I like bbox

Reply:
Changed back